src/relayserver/ReputationStoreManager.ts

```diff
@@ -0,0 +1,89 @@
+import AsyncNedb from 'nedb-async'
+
+import { LoggerInterface } from '../common/LoggerInterface'
+import { ReputationChange, ReputationEntry } from './ReputationEntry'
+import { Address } from '../relayclient/types/Aliases'
+
+export const REPUTATION_STORE_FILENAME = 'reputation_store.db'
+
+export class ReputationStoreManager {
+  private readonly txstore: AsyncNedb<any>
+  private readonly logger: LoggerInterface
+
+  constructor ({ workdir = '/tmp/test/' }, logger: LoggerInterface) {
+    this.logger = logger
+    const filename = `${workdir}/${REPUTATION_STORE_FILENAME}`
```

Comment:
I think we should support "inmemory" mode , when workdir==null

Reply:
not needed outside tests and dev mode already removes all the files, so it is a feature creep IMO